public/login_app.php

```diff
@@ -1,10 +0,0 @@
-<?php
-
-// Auto login from app uses token. Standard login from app uses password.
-$user = request()->post('u');
-$pass = request()->post('p');
-$token = request()->post('t');
-
-$response = authenticateForConnect($user, $pass, $token);
```

Comment:
This file was used by the DLL prior to version 0.79. As the minimum supported version is 1.x, and I didn't see any recent references to the URL in the server logs, I believe this is safe to remove.